### PR TITLE
Add detailed timeout descriptions on Riak API call in CS

### PR DESCRIPTION
**Pull request by kuenishi**
Addresses https://github.com/basho/riak_cs/issues/1102 .

### CODE REVIEW

File: source/languages/en/riakcs/cookbooks/configuration/configuration-reference.md [hunk 1]
```diff
@@ -741,6 +741,79 @@ input (which is the default).</td>
...
 </table>
 
+## Timeouts on each Riak call
+
+As Riak CS stores all data in underlying Riak, Riak CS process
+communicates to Riak over protocol buffers API. This is typical remote
+call - depending system requirements, this timeout could be configured
+to avoid unnecessary timeouts.
+
+{{#1.5.2-}}
```
Comment (ksauzz) [inline source/languages/en/riakcs/cookbooks/configuration/configuration-reference.md]: As this means 'lesser than 1.5.2, exclusive', it should be `{{#1.5.3-}}`

File: source/languages/en/riakcs/cookbooks/configuration/configuration-reference.md [hunk 2]
```diff
@@ -741,6 +741,79 @@ input (which is the default).</td>
...
+
+This example changes default timeout of each primitive put and get
+call to Riak, from default 60 seconds to 5 seconds to improve latency,
+with the cost of more frequent timeout error possibility. This example
+also changed default timeout of storage calculation call timeout on
+each bucket, from default 60 seconds to 20 minutes, considering the
+duration of storage calculation proportionally increases to the number
+of keys in a bucket.
+
+{{/1.5.2-}}
```
Comment (ksauzz) [inline source/languages/en/riakcs/cookbooks/configuration/configuration-reference.md]: This should be 1.5.3 as well.

**Comment by ksauzz** [comment]
Nice work. Although I fixed cosmetic things instead of you, +1 from me.

**Comment by kuenishi** [comment]
Thanks, I'll wait for a review from docs team or other.

**Comment by laurenrother** [comment]
Looks good! :cake:

**Pull request merged**